code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { DETAILS_URL } from "utils/urls";
+import './moviedetails.css'
+import { useHistory } from "react-router-dom";
+// import { FaChevronCircleLeft } from 'react-icons/fa' <FaChevronCircleLeft />
+import { AiOutlineArrowLeft } from "react-icons/ai";
+
+const MovieDetails = () => {
+const [movieDetails, setMovieDetails] = useState({});
+const { movieId } = useParams();
+const history = useHistory();
+
+useEffect(() => {
+    fetch(DETAILS_URL(movieId))
+    .then((res) => res.json())
+    .then((json) => {
+        setMovieDetails(json);
+        console.log("DETAILS", json);
+    });
+}, [movieId]);
+
+ 
+return (
+    <section className="wrapper-details">
+        <button className="backbutton" onClick={() => history.goBack()}> <AiOutlineArrowLeft /> <h3 className="button-text">Movies</h3></button>
```

Comment:
We can see that you have used the AiOutlineArrowLeft but it is currently invisible in your app., maybe it needs to be filled?